chat-plugins/info.js

```diff
@@ -1340,7 +1340,7 @@ exports.commands = {
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(
 			`New to competitive Pok&eacute;mon?<br />` +
-			`- <a href="http://www.smogon.com/forums/posts/6774481/">Beginner's Guide to Pok&eacute;mon Showdown</a><br />` +
+			`- <a href="http://www.smogon.com/forums/threads/the-beginners-guide-to-pokémon-showdown.3496279/">Beginner's Guide to Pok&eacute;mon Showdown</a><br />` +
```

Comment:
This isn't how PS! links to smogon threads and posts. The title should not be a part of the link (``/threads/123456/``).